docs/installation/centos.md

```diff
@@ -12,153 +12,185 @@ parent = "smn_linux"
 
 Docker is supported on the following versions of CentOS:
 
-- [*CentOS 7 (64-bit)*](#installing-docker-centos-7)
-- [*CentOS 6.5 (64-bit)*](#installing-docker-centos-6.5) or later
+* CentOS 7.X 
+* CentOS 6.5 or higher 
+
+Installation on other binary compatible EL6/EL7 distributions such as Scientific
+Linux might succeed, but Docker does not test or support Docker on these
+distributions.
+
+This page instructs you to install using Docker-managed release packages and
+installation mechanisms. Using these packages ensures you get the latest release
+of Docker. If you wish to install using CentOS-managed packages, consult your
+CentOS documentation.
+
+## Prerequisites
+
+Docker requires a 64-bit installation regardless of your CentOS version. Also,
+your kernel must be 3.10 at minimum. CentOS 7 runs the 3.10 kernel, 6.5 does
+not. We make an exception for CentOS 6.5. To run Docker on
```

Comment:
concise wording is fine, as long as it's clear.

I would be fine with "latest centos6 release" or similar. The specification of `6.x` is due to the fixes included for docker/devicemapper/etc. It is cumbersome, but it is due to many customers and deployments pinning a version of RHEL or centos (e.g. 6.4) and refusing to update. Sometimes these deployments are willing to cherry-pick RPMS (e.g. kernel) from a newer release, but that is largely at their own risk. By Docker specifying the minor version, it sets the bar, and narrows the supportability of pinned deployments.